test/harness/jpype/collection/CollectionTest.java

```diff
@@ -0,0 +1,17 @@
+package jpype.collection;
```

Comment:
is this file used elsewhere?

Reply:
This was a left over from the user report.  I think I should make a test that passes a Python list and make sure I get an array list out the other side.  Same with map